test/cmd/templates.sh

```diff
@@ -22,6 +22,17 @@ os::cmd::expect_success 'oc get templates'
 os::cmd::expect_success 'oc get templates ruby-helloworld-sample'
 os::cmd::expect_success 'oc get template ruby-helloworld-sample -o json | oc process -f -'
 os::cmd::expect_success 'oc process ruby-helloworld-sample'
+os::cmd::expect_success_and_text 'oc process ruby-helloworld-sample -o template --template "{{.kind}}"'    "List"
```

Comment:
Yes. Like in `oc get` it is not shown in help but still works. I suppose it is deprecated in favor of `-o go-template`.